docs/_includes/dependenciesx_list.md

```diff
@@ -1,3 +1,7 @@
+{% if include.component == "fortify" %}
```

Comment:
I think this file is auto generated
but probably won't hurt if it's modify and will just get overwritten

Reply:
As far as I understand it is generated through some .py tool. Leaving it until told otherwise.

Reply:
ACEX one is not generated through Travis CI, because I'd have to clone the whole repo, would take unnecessary time. You can pass `--acex` to the tool and provide it with a path to ACEX, but since it isn't merged yet, simpler to do it like that.